### PR TITLE
John conroy/deploy to chromatic

### LINKED ISSUE #1919: Deploy storybook (to github pages?)
My guess is that a good way to do this would be to have the deployment script also build and push the storybook to a `gh-pages` branch, but pushing to s3 is also a possibility.

If we do use gh-pages, could we re-write that branch in place, rather than adding new commits to the repo history?

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #1919.

If you log into `https://www.chromatic.com/` via GitHub, you should have access to manage chromatic. Access should mirror that of this repo. Let me know if you'd prefer to register through another means.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,8 @@ npm run lint:fix
 ### Storybook
 To start storybook locally run `npm run storybook`.
 
+To view the production storybook visit [chromatic](https://main--60e472b3f21a0d003bd71f2f.chromatic.com). To view the latest published storybook on a given branch, visit `https://<branch>--60e472b3f21a0d003bd71f2f.chromatic.com`. Replace <branch> with the designated branch.
```
Comment (mccalluc) [inline README.md:110]: Link is 404 for me.

Comment (mccalluc) [inline README.md:110]:
```suggestion
To view the production storybook visit [chromatic](https://main--60e472b3f21a0d003bd71f2f.chromatic.com). To view the latest published storybook on a given branch, visit `https://<branch>--60e472b3f21a0d003bd71f2f.chromatic.com`. Replace `<branch>` with the designated branch.
```

**Review by mccalluc — COMMENTED** [review]
Getting a 404: Fine to merge when that's fixed.

File: README.md (hunk 1 above)
Comment (john-conroy) [reply README.md:110]: You shouldn't get a 404 once it's merged.

**Pull request merged**